lib/extract_field/login_successful/HeaderLoginSuccessfulParser.js

```diff
@@ -0,0 +1,18 @@
+const { ILoginSuccessfulParser } = require('./ILoginSuccessfulParser');
+const { DEFAULT_LOGIN_SUCCESSFUL_HEADER_NAME, DEFAULT_LOGIN_SUCCESSFUL_HEADER_VALUE } = require('../../utils/constants');
+
+class HeaderLoginSuccessfulParser extends ILoginSuccessfulParser {
```

Comment:
all these `LoginSuccessfulParsers` define how **we** understand if a login was successful right? not how we communicate if the login was success or not in the enforcer activities

Reply:
What's the difference?